### PR TITLE
Cureforcancer

### PR DESCRIPTION
**Pull request by Robertha89**
Now we can cure cancer with chems! No need to surgery everything.
:cl: ppi
rscadd: Adds a cure for brain cancer.
/:cl:

### CODE REVIEW

File: code/modules/reagents/newchem/medicine.dm [hunk 1]
```diff
@@ -1075,4 +1075,50 @@ datum/reagent/medicine/syndicate_nanites/on_mob_life(mob/living/M)
...
+	if(prob(10))
+		M.adjustToxLoss(4)
+	if(prob(10))
+		M.adjustBruteLoss(4)
+	if(prob(5))
+		M.fakevomit()
+	if(prob(1))
+		M.fakepoop()
+	if((world.time - M.last_movement) >= 0)
+		M.halloss++ //the more you walk, the more pain you are in
```
Comment (Fox-McCloud) [inline code/modules/reagents/newchem/medicine.dm]: Halloss is deprecated entirely by staminaloss.

**Comment by Fox-McCloud** [comment]
No. just. No.

**Comment by TheDZD** [comment]
No.
Also you have merge commits in a brand new, stop it with those.

**Comment by Magikarpet** [comment]
is this a stealth shadowling nerf? From what I see it looks like a stealth shadowling nerf

**Comment by Robertha89** [comment]
yeah, gonna fix that, was not supposed to push this untill it was fixed.

**Comment by TheDZD** [comment]
@Magikarpet It adds a reagent that can cure shadowling enthralling.

**Pull request closed without merging**